### PR TITLE
Fix: deal properly with tc qdisc show returning "noqueue"

### PR DESCRIPTION
**Pull request by therc**
Fixes issue #19237

### CODE REVIEW

File: pkg/util/bandwidth/linux.go [hunk 1]
```diff
@@ -224,6 +224,12 @@ func (t *tcShaper) interfaceExists() (bool, string, error) {
 	if len(value) == 0 {
 		return false, "", nil
 	}
+	// Newer versions of tc and/or the kernel return the following instead of nothing:
+	// qdisc noqueue 0: root refcnt 2
+	fields := strings.Split(value, " ")
```
Comment (thockin) [inline pkg/util/bandwidth/linux.go]: We should use `strings.Fields(value)` more - it is multi-whitespace safe and more descriptive.

**Comment by thockin** [comment]
minor nit, then LGTM.  I assume this works for you?

**Comment by thockin** [comment]
ok to test

**Comment by therc** [comment]
Yes, it's working for me. It took a bit to rework my turnup procedure to include arbitrary binaries.

**Comment by thockin** [comment]
LGTM!  Thanks.

We might want to cherry pick this back into the 1.1 branch - I can do that for you

**Comment by thockin** [comment]
Can you please squash this?

**Comment by therc** [comment]
Done

**Comment by eparis** [comment]
@k8s-bot unit test this

Not sure why apiserver would be flaking here....

**Comment by iterion** [comment]
@thockin :+1: on cherry picking this into 1.1, would love to be able to run the stable release instead of head ;)

**Comment by eliaslevy** [comment]
:+1: on back porting this into 1.1.

**Comment by eparis** [comment]
@k8s-bot unit test this

**Comment by thockin** [comment]
@k8s-bot test this
@k8s-bot unit test this

**Pull request merged**